modules/programs/firefox.nix

```diff
@@ -206,7 +206,14 @@ let
     else if versionAtLeast config.home.stateVersion "19.09" then
       package.override (old: {
         cfg = old.cfg or { } // fcfg;
-        extraPolicies = (old.extraPolicies or { }) // cfg.policies;
+        extraPolicies = (old.extraPolicies or { }) // {
+          ExtensionSettings = listToAttrs (map (lang:
+            nameValuePair "langpack-${lang}@firefox.mozilla.org" {
+              installation_mode = "normal_installed";
+              install_url =
+                "https://releases.mozilla.org/pub/firefox/releases/${cfg.package.version}/linux-x86_64/xpi/${lang}.xpi";
+            }) cfg.languagePacks);
+        } // cfg.policies;
```

Comment:
Any reason why this merging code (which should probably be recursive, too) is used instead of just setting the `policies` option?